src/ecdc/ecdc.c

```diff
@@ -825,6 +835,26 @@ ecdc_alloc_list_command(struct ecdc_console * console,
 }
 
 
+char const * ecdc_replace_prompt(struct ecdc_console *console, char const *prompt)
+{
+    if ((NULL == console) || (NULL == prompt))
+    {
+        return NULL;
+    }
+
+    // If the prompt was previously not set, it will be NULL which will not
+    // have any ill side effects when calling free
+    free(console->prompt);
+
+    console->prompt = strdup(prompt);
+    return console->prompt;
+}
+
+void ecdc_free_prompt(struct ecdc_console *console)
```

Comment:
Update format to be consistent with the rest of the file
```
void
ecdc_free_prompt(struct ecdc_console *console)
{
    // ...
}
```

Reply:
Done.